<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sctiptDialog</class>
 <widget class="QDialog" name="sctiptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>运动脚本管理</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="_2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>动作流程</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>循环次数</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="RspinBox">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="AddRbtn">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="script_management.qrc">
                 <normaloff>:/images/Resources/images/add.png</normaloff>:/images/Resources/images/add.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>升降速度</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="VhorizontalSlider">
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>10</string>
                 </property>
                 <property name="margin">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="AddVbtn">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="script_management.qrc">
                 <normaloff>:/images/Resources/images/add.png</normaloff>:/images/Resources/images/add.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>位移</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="LhorizontalSlider">
               <property name="minimum">
                <number>-1500</number>
               </property>
               <property name="maximum">
                <number>1500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="margin">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="3">
              <widget class="QPushButton" name="AddLbtn">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="script_management.qrc">
                 <normaloff>:/images/Resources/images/add.png</normaloff>:/images/Resources/images/add.png</iconset>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>旋转分度</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="TspinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="AddTbtn">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="script_management.qrc">
                 <normaloff>:/images/Resources/images/add.png</normaloff>:/images/Resources/images/add.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>19</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,10,1,0">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>文件名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filenameEdit"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="newScriptBtn">
               <property name="text">
                <string>新建</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>管理</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QComboBox" name="script_cb"/>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>运动脚本</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="deleteBtn">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="scriptEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>R(循环次数)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>V(升降速度)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>L(垂直位移)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>T(旋转分度)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>RspinBox</tabstop>
  <tabstop>AddRbtn</tabstop>
  <tabstop>AddVbtn</tabstop>
  <tabstop>LhorizontalSlider</tabstop>
  <tabstop>AddLbtn</tabstop>
  <tabstop>TspinBox</tabstop>
  <tabstop>AddTbtn</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>script_cb</tabstop>
  <tabstop>scriptEdit</tabstop>
  <tabstop>deleteBtn</tabstop>
 </tabstops>
 <resources>
  <include location="script_management.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>VhorizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LhorizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
